google/gax/__init__.py

```diff
@@ -33,7 +33,7 @@
 import collections
 
 
-__version__ = '0.12.5'
+__version__ = '0.12.6'
```

Comment:
I'd feel more comfortable calling this `0.13.0`, unless you have strong reason to believe that there's nothing breaking in this change to gRPC `1.0.0`.